msg/distance_sensor.msg

```diff
@@ -5,6 +5,7 @@ float32 min_distance		# Minimum distance the sensor can measure (in m)
 float32 max_distance		# Maximum distance the sensor can measure (in m)
 float32 current_distance	# Current distance reading (in m)
 float32 covariance		# Measurement covariance (in m), 0 for unknown / invalid readings
+int8 signal_strength		# Signal strength in percent (0...100%) or -1 if unknown
```

Comment:
There are sensing modalities that do not use reflected signal strength, eg stereo vision, so this needs to be a generic description.

Reply:
0-255 for full range of uint8 maybe?

Reply:
> Please change to: int8 quality	# measurement quality in percent (0...100%) or -1 if unknown. A value of 0 indicates an invalid measurement.

Changed.

> 0-255 for full range of uint8 maybe?

We're using int8 instead of uint8 to have the capability to indicate unknown signal quality (i.e. the driver does not support this) with a -1. The signal quality also does not need to be super precise, it is a ballpark-estimate in any cas.e..

Reply:
@philipoe Agree. Lets use 0-100.